jgiven-gradle-plugin/src/main/java/com/tngtech/jgiven/gradle/internal/AbstractJGivenReportImpl.java

```diff
@@ -18,13 +18,13 @@ public AbstractJGivenReportImpl( String name, Task task, String relativeEntryPat
     }
 
     @Override public void configure( ReportGenerator generator ) {
-        generator.setTargetDirectory( getDestination() );
-        generator.setFormat( getFormat() );
-        generator.getConfig().setCustomCssFile( getCustomCssFile() );
-        generator.getConfig().setCustomJsFile( getCustomJsFile() );
-        generator.getConfig().setExcludeEmptyScenarios( isExcludeEmptyScenarios() );
-        if (getTitle() != null) {
-            generator.getConfig().setTitle( getTitle() );
+        generator.addFlag( "--targetDir=" + getDestination() );
```

Comment:
I don't really like introducing String constants here. Why not keeping the old methods? Maybe it would be useful to separate the configuration option logic into a class that is separated from the generator?